src/main/typescript/wcardinal/ui/d-slider.ts

```diff
@@ -114,7 +117,7 @@ export abstract class DSlider<
 		};
 	}
 
-	protected newValues( options?: OPTIONS ): void {
+	protected initValues( options?: OPTIONS ): { min: number, max: number, value: DSliderValueOptions } {
```

Comment:
Please revisit the method name.

Reply:
This method for initializing min value, max value and current value.
Then, DSliderLabel, DSliderValue will be created by:
```javascript
this._value = this.newValue( initializedValues.value );
this._min = this.newLabel( initializedValues.min );
this._max = this.newLabel( initializedValues.max );
```
Would you tell what should I do?

Reply:
I thought it is extracting the min / max / current values from the given options. Not initializing something.

Reply:
In some case it't initializing. 
For example, min / max / current values are not included by options.
Anyway, `extracting` is OK.
Change to `extractValues` . Are you OK?

Reply:
👍